tutorials/katacoda/thanos/2-lts/step1.md

```diff
@@ -0,0 +1,138 @@
+# Step 1 - Initial Prometheus Setup
+
+Thanos is a set of components that adds high availability to Prometheus installations by the cross-cluster federation, unlimited storage, and global querying across clusters.
+
+Thanos builds upon vanilla Prometheus instances which states it seamlessly integrates into existing Prometheus setups.
+
+In this tutorial, we will be using one Prometheus Server in the eu1 region.
+
+Let's start this initial Prometheus setup, ready?
+
+
+## Generate Artifical Metric Data
+
+Execute the following command:
+
+```
+docker run quay.io/thanos/thanosbench:docker-2019-10-04-a84ac8e block plan -p realistic-k8s-2d-small --labels 'cluster="one"' --max-time 2019-10-18T00:00:00Z
```

Comment:
@bwplotka adding image `dockerenginesonia/thanosbench:1.0` seems to work fine, (as Povilas suggested me to push own thanosbench image into the repository), the current image `quay.io/thanos/thanosbench:docker-2019-10-04-a84ac8e` is giving error:
```
Error parsing commandline arguments: expected command but got "block"
thanosbench: error: expected command but got "block"
```

Reply:
And can you please tell me how to further share this `dir` with the prometheus?
(because the prometheus container is not yet started 🤔)

Reply:
Marking resolved !!!